apps/meteor/server/services/federation/infrastructure/rocket-chat/adapters/Settings.ts

```diff
@@ -176,6 +176,17 @@ export class RocketChatSettingsAdapter {
 			section: 'Matrix Bridge',
 		});
 
+		await settingsRegistry.add('Federation_Matrix_serve_well_known', false, {
```

Comment:
Why not true by default? If reverse proxy intercepts it to serve it won't pass to the server anyways.